hadoop-ozone/recon-codegen/src/main/java/org/hadoop/ozone/recon/schema/SchemaVersionTableDefinition.java

```diff
@@ -48,11 +53,20 @@ public SchemaVersionTableDefinition(DataSource dataSource) {
 
   @Override
   public void initializeSchema() throws SQLException {
-    Connection conn = dataSource.getConnection();
-    dslContext = DSL.using(conn);
+    try (Connection conn = dataSource.getConnection()) {
+      dslContext = DSL.using(conn);
```

Comment:
dslContext outside of try may be invalid, as conn will be closed. So having as member variable may be risky,

Reply:
Done!